src/sqlfluff/dialects/dialect_sparksql.py

```diff
@@ -2685,3 +2687,24 @@ class DescribeDetailStatementSegment(BaseSegment):
             Ref("TableReferenceSegment"),
         ),
     )
+
+
+class GenerateManifestFileStatementSegment(BaseSegment):
+    """A statement to `GENERATE` manifest files for a Delta Table.
+
+    https://docs.delta.io/latest/delta-utility.html#generate-a-manifest-file
+    """
+
+    type = "generate_manifest_file_statement"
+
+    match_grammar: Matchable = Sequence(
+        "GENERATE",
+        Ref("BareFunctionSegment"),
```

Comment:
Why are you using `BareFunctionSegment` instead of hardcoding `"symlink_format_manifest"` here? Are the other bare functions valid here?

If so should we include in your test cases?
If not, should we just hardcode this to `"symlink_format_manifest"` here?

Reply:
I'll update to have it be hardcoded